airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1407,6 +1408,15 @@ class ContactsListMemberships(ContactsAllBase, ABC):
     filter_field = "showListMemberships"
     filter_value = True
 
+    @property
+    def updated_at_field(self):
+        """Name of the field associated with the state"""
+        return "timestamp"
+
+    @property
+    def cursor_field_datetime_format(self):
+        return "x"
```

Comment:
Can we add a reference to docs about this?

Reply:
+1